common/components/ui/Help.tsx

```diff
@@ -5,13 +5,20 @@ import './Help.scss';
 type Size = 'x1' | 'x2' | 'x3';
 
 interface Props {
-  link: string;
+  link?: string;
   size?: Size;
+  tooltip?: string;
 }
 
-const Help = ({ size = 'x1', link }: Props) => {
+const Help = ({ size = 'x1', link, tooltip }: Props) => {
   return (
-    <a href={link} className={`Help Help-${size}`} target="_blank" rel="noopener noreferrer">
+    <a
+      href={link}
+      className={`Help Help-${size}`}
+      target="_blank"
+      rel="noopener noreferrer"
+      title={tooltip}
```

Comment:
We'll want to make an actual tooltip instead, I think this behavior is not very intuitive since it still renders and behaves like a button, and title text takes a few seconds on most browsers.

Reply:
See `common/components/Header/OnlineStatus.tsx` for an example of our tooltips, requires a React component and a css mixin to get working.

Reply:
Changed. :)